conftest.py

```diff
@@ -801,17 +803,10 @@ def global_dragon_teardown() -> None:
     """
     if test_launcher != "dragon" or CONFIG.dragon_server_path is None:
         return
-    exp_path = os.path.join(test_output_root, "dragon_teardown")
-    os.makedirs(exp_path, exist_ok=True)
-    exp: Experiment = Experiment("dragon_shutdown", exp_path=exp_path, launcher=test_launcher)
-    rs = exp.create_run_settings("sleep", ["0.1"])
-    model = exp.create_model("dummy", run_settings=rs)
-    exp.generate(model, overwrite=True)
-    exp.start(model, block=True)
-
-    launcher: DragonLauncher = exp._control._launcher
-    launcher.cleanup()
-    time.sleep(5)
+    logger.debug(f"Tearing down Dragon infrastructure, server path: {CONFIG.dragon_server_path}")
+    dragon_connector = DragonConnector()
+    dragon_connector.ensure_connected()
```

Comment:
It seems like we might get a benefit similar to shared databases if we avoid destroying and re-creating a dragon environment for every test (less likely to fail, faster test runs, etc.). Is there any reason not to re-use?

Reply:
We use the same for almost all tests, but I wanted to test that starting it with different paths was working.